FFCDemoPaymentService/appsettings.json

```diff
@@ -14,16 +14,12 @@
     "ScheduleQueueName": "schedule",
     "ScheduleQueueEndpoint": "http://localhost:9324",
     "ScheduleQueueUrl": "http://localhost:9324/queue/schedule",
-    "ScheduleQueueRegion": "eu-west-2",
-    "ScheduleAccessKeyId": "",
-    "ScheduleAccessKey": "",
     "CreateScheduleQueue": true,
     "PaymentQueueName": "payment",
     "PaymentQueueEndpoint": "http://localhost:9324",
     "PaymentQueueUrl": "http://localhost:9324/queue/payment",
-    "PaymentQueueRegion": "eu-west-2",
-    "PaymentAccessKeyId": "",
-    "PaymentAccessKey": "",
-    "CreatePaymentQueue": true
+    "CreatePaymentQueue": true,
```

Comment:
Something to come back to in future rather than address in the PR, but if this Message settings was more hierarchical than flat i.e. `Messaging: { scheduleConfig: {}, PaymentConfig: {} }`
The SqsConfig type could be used for the scheduleConfig and PaymentConfig settings, which would remove the need for the very similar types PaymentMap and ScheduleMap.

Reply:
Yes good observation, this would be an improvement.